.eslintrc.json

```diff
@@ -1,22 +1,32 @@
-{
+{
   "env": {
-    "es6": true
-  },
-  "extends": ["prettier"],
-  "globals": {
-    "Atomics": "readonly",
-    "SharedArrayBuffer": "readonly"
-  },
-  "parser": "babel-eslint",
-  "parserOptions": {
-    "ecmaFeatures": {
-      "jsx": true
-    },
-    "ecmaVersion": 2018,
-    "sourceType": "module"
-  },
-  "plugins": ["prettier", "react"],
+    "browser": true,
+    "es6": true
+  },
+  "extends": [
+    "eslint:recommended",
+    "plugin:react/recommended",
+    "plugin:prettier/recommended",
+    "prettier/react"
+  ],
+  "globals": {
+    "Atomics": "readonly",
+    "SharedArrayBuffer": "readonly"
+  },
+  "parser": "babel-eslint",
+  "parserOptions": {
+    "ecmaFeatures": {
+      "jsx": true
+    },
+    "ecmaVersion": 2019,
+    "sourceType": "module"
+  },
+  "plugins": ["prettier", "react"],
   "rules": {
-    "prettier/prettier": "warn"
-  }
-}
+    "eqeqeq": "error",
```

Comment:
All checks that are not required by prettier pre-commit check should show up as warn instead of error to avoid confusion when pre-commit check passes with linting errors

Reply:
Now at least eqeqeq shows up as warning, but there might be other rules in eslint:recommended that are preferred as warnings instead of errors; they will be changed as needed.